packages/central-server/src/database/models/User.js

```diff
@@ -30,6 +46,35 @@ class UserRecord extends CommonUserRecord {
 }
 
 export class UserModel extends CommonUserModel {
+  meditrakConfig = {
+    // only sync id and first and last name
+    ignorableFields: [
+      'gender',
+      'creation_date',
+      'employer',
+      'position',
+      'mobile_number',
+      'password_hash',
+      'password_salt',
+      'verified_email',
+      'profile_image',
+      'primary_platform',
+      'preferences',
+      'full_name', // ignore this because it isn't a real field, it's a derived field
+    ],
+    translateRecordForSync: record => {
+      const { email, ...restOfRecord } = record;
+      const isInternal = INTERNAL_USERS.includes(email) || INTERNAL_EMAIL_REGEXP.test(email);
+
+      // Flag internal users. These will be filtered out in meditrak-app
+      if (isInternal) {
+        return { ...restOfRecord, internal: true };
+      }
+      return restOfRecord;
+    },
+    minAppVersion: '1.14.142',
```

Comment:
```suggestion
    minAppVersion: '1.14.143',
```
Might be worth bumping the version as part of this ticket too. If we did accidentally release this without a version bump it would create a few problemos :sweat_smile:

Reply:
Will this mean we can't test it?